revChatGPT/Exceptions.py,sha256=70Dmggf9AQGBKnfg0NJJmyKugBUig42THIs6OD5jeMA,632
revChatGPT/__init__.py,sha256=47DEQpj8HBSa-_TImW-5JCeuQeRkm5NMpJWZG3hSuFU,0
revChatGPT/__main__.py,sha256=69RodheUQABmGul2-CxzhJ0tSXHx7pmkxya3btruxUU,6124
revChatGPT/revChatGPT.py,sha256=0cLVCsc2vh3skv1JC2bv-G7CSpkHdkiBS32k9sPOW8k,20671
revChatGPT-0.0.44.dist-info/LICENSE,sha256=gXf5dRMhNSbfLPYYTY_5hsZ1r7UU1OaKQEAQUhuIBkM,18092
revChatGPT-0.0.44.dist-info/METADATA,sha256=JCJu7xkWYKcfzBLiR16AosQUWxPrIGplXQ6sxHC_bXg,2920
revChatGPT-0.0.44.dist-info/WHEEL,sha256=2wepM1nk4DS4eFpYrW1TTqPcoGNfHhhO_i5m4cOimbo,92
revChatGPT-0.0.44.dist-info/entry_points.txt,sha256=XWRsRhbycEK0s0FlJsWUcWPSuZ5Gr0InFuJdEiKeUiE,56
revChatGPT-0.0.44.dist-info/top_level.txt,sha256=6O3ay3adff-Qrq0tGYaEe_XXhvCZ5vGC9AshCmZzRMA,11
revChatGPT-0.0.44.dist-info/RECORD,,
